tests/lib/outputs/finding_test.py

```diff
@@ -145,7 +152,8 @@ def test_generate_output_aws(self):
         check_output.status_extended = "mock_status_extended"
         check_output.muted = False
         check_output.check_metadata = mock_check_metadata(provider="aws")
-        check_output.resource = {}
+        check_output.resource = {"metadata": "mock_metadata"}
+        check_output.compliance = {"mock_compliance_key": "mock_compliance_value"}
```

Comment:
Could we change this to ensure that the compliance field follow the same format as the original?
![Screenshot 2025-04-04 at 13 56 26](https://github.com/user-attachments/assets/01989b97-346c-46c3-960b-aab3eecc28e4)
`"compliance": {
            "CIS-2.0": [
                "1.12"
            ],
            "CIS-3.0": [
                "1.12"
            ],
            "ENS-RD2022": [
                "op.acc.2.gcp.rbak.1"
            ],
            "MITRE-ATTACK": [
                "T1098"
            ]
        }`

Reply:
Sure, done!